.github/workflows/update-urls.yml

```diff
@@ -27,7 +27,9 @@ jobs:
         run: |
           cd cli
           mvn -B -ntp -Dstyle.color=always install
-          mvn -B -ntp -Dstyle.color=always exec:java -Dexec.mainClass="com.devonfw.tools.ide.url.UpdateInitiator" -Dexec.args="../ide-urls PT5H30M"
+          mvn -B -ntp -Dstyle.color=always exec:java -Dexec.mainClass="com.devonfw.tools.ide.url.UpdateInitiator" -Dexec.args="../ide-urls PT5H30M" | tee log.txt
+          sed -n '/ERROR REPORT FROM/,/ERROR REPORT ENDS/p' log.txt > error_report.txt
+          cat error_report.txt
```

Comment:
If we have Java to collect all the `errorReport`, then why can't this directly write the `error_report.txt` at the end instead of writing all logs to disc and then do `sed` magic filtering the `error report` out of the log that we have produced ourselves in Java in a structured way?